common/test/acceptance/pages/lms/discussion.py

```diff
@@ -733,3 +733,11 @@ def get_new_post_preview_text(self):
         """
         self.wait_for_element_visibility(".wmd-preview > div", "WMD preview pane has contents", timeout=10)
         return self.q(css=".wmd-preview").text[0]
+
+    def get_preview_value(self):
+        """
+        Waits for content to appear
+        Get the rendered preview of the contents of the Discussions new post editor
```

Comment:
Please add description here why we are not using `get_new_post_preview_value` or `get_new_post_preview_text` here. One reason which comes into my mind is we don't want to look for any direct children of element with class `wmd-preview` which other two methods are looking for.